contracts/contracts/validator-registry/VanillaRegistry.sol

```diff
@@ -142,13 +143,21 @@ contract VanillaRegistry is IVanillaRegistry, VanillaRegistryStorage,
         _unstake(blsPubKeys);
     }
 
-    /* 
-     * @dev Withdraws ETH on behalf of one or multiple validators via their BLS pubkey.
-     * @param blsPubKeys The BLS public keys to withdraw.
-     */
+    /// @dev Allows owner to withdraw ETH on behalf of one or multiple validators via their BLS pubkey.
+    /// @param blsPubKeys The BLS public keys to withdraw.
+    /// @dev msg.sender must be the withdrawal address for every provided validator pubkey as enforced in _withdraw.
     function withdraw(bytes[] calldata blsPubKeys) external
-        onlyExistentValidatorRecords(blsPubKeys) onlyWithdrawalAddress(blsPubKeys) whenNotPaused() {
-        _withdraw(blsPubKeys);
+        onlyExistentValidatorRecords(blsPubKeys) whenNotPaused() {
```

Comment:
Removed `onlyWithdrawalAddress` modifier here since the check now resides in `_withdraw` to accommodate two different usages of the `_withdraw` function